rules/streamline-approvals/rule.md

```diff
@@ -0,0 +1,53 @@
+---
+type: rule
+archivedreason: 
+title: Do you know how to streamline approvals?
+guid: 4f425961-3b0e-4b1b-bea4-48bab98f5c05
+uri: streamline-approvals
+created: 2023-08-08T13:00:00.0000000Z
+authors:
+- title: Piers Sinclair
+  url: https://ssw.com.au/people/piers-sinclair
+related: 
+  - bench-master
+redirects: []
+
+---
+
+In small companies, a single key stakeholder often approves everything, ensuring alignment with the organizational vision and strategy. However, as the company grows, this can lead to bottlenecks. To address this, one strategy is to create an employee responsibility which lists the people responsible for a specific approval. This list creates a clearly communicated system for [cross-authorization](https://www.ssw.com.au/rules/purchase-please/#cross-authorization).
+
+<!--endintro-->
+
+## Assigning employee responsibilities
+1. Identify areas for improvement. Look for approvals causing bottlenecks.
+   e.g. delays in approval for induction completion.
+2. Once identified, compile a list of subject matter experts who will be responsible for that approval.
+3. Assign each person a priority, with the original approver having the lowest priority to minimize their involvement.
+4. Induct each person by having them shadow the original approver for at least three approval processes, ensuring they understand the goal.
+5. Document the list in a central repository, naming it {{ APPROVAL }} Masters. These individuals are now responsible for that approval.
+e.g. the Induction Masters are responsible for induction approval.
+6. Update the standard so that instead of contacting the original approver, individuals contact the list in order until they reach any 2 of the {{ APPROVAL }} Masters. If they get approval 2 approval masters, they are good to go!
```

Comment:
```suggestion
6. Update the standard so that instead of contacting the original approver, individuals contact the list in order until they reach any 2 of the {{ APPROVAL }} Masters. If they get approval from 2 approval masters, they are good to go!
```